docs/setup/settings.asciidoc

```diff
@@ -325,11 +342,16 @@ default is `true`.
 `server.socketTimeout:`:: *Default: "120000"* The number of milliseconds to wait before closing an
 inactive socket.
 
-`server.ssl.certificate:` and `server.ssl.key:`:: Paths to the PEM-format SSL
-certificate and SSL key files, respectively.
+`server.ssl.certificate:` and `server.ssl.key:`:: Paths to a PEM-encoded X.509 certificate and its private key, respectively. These are used
+when enabling SSL for inbound requests from web browsers to the Kibana server.
 
-`server.ssl.certificateAuthorities:`:: List of paths to PEM encoded certificate
-files that should be trusted.
+NOTE: These settings cannot be used in conjunction with `server.ssl.keystore.path`.
```

Comment:
If this note should be indented under the server.ssl.certificate and server.ssl.key description:
```suggestion
+
--
NOTE: These settings cannot be used in conjunction with `server.ssl.keystore.path`.

--
```